generative_models/AAE_PyTorch/README.md

```diff
@@ -0,0 +1,100 @@
+# Adversarial Autoencoder
+## Usage
+```bash
+> python main.py --arguments
```

Comment:
your usage script should be a valid one, so replace --arguments with actual example of some arguments

Reply:
Done